### DISCUSSION TITLE
Best position for the cameras

### DISCUSSION THREAD

**Discussion by Touville01** [roboflow/supervision#1217] [Q&A]
Hi everyone,

I'm currently using YOLO models and Supervision scripts for person detection and tracking, and I'm looking for some advice on camera placement.

Optimizing Camera Angles for Person Detection:

What are the best angles and positions for cameras to optimize person detection with YOLO models?
Are there specific guidelines or best practices for camera placement to improve the accuracy and reliability of detections using YOLO and Supervision scripts?
Any tips or recommendations on how to achieve the best results would be greatly appreciated!

Thank you!

**Comment by SkalskiP** [comment]
Hi @Touville01 👋🏻 You will use models pre-trained on the COCO dataset or fine-tune on your custom dataset?

**Reply by Touville01** [reply]
Hi, I will use pre-trained models !

**Reply by SkalskiP** [reply]
The COCO dataset has almost no images of people taken from above, which significantly reduces detection quality if the camera is mounted on the ceiling and looking straight down. Try to position the camera to view the object more from the side. Additionally, I recommend watching [this](https://www.youtube.com/watch?v=hAWpsIuem10&t=983s) part of my video where I discuss camera placement.
